compiler/main/driver.cpp

```diff
@@ -1584,6 +1597,47 @@ static void checkUnsupportedConfigs(void) {
   }
 }
 
+static void checkRuntimeBuilt(void) {
+  // no need for a runtime to be built for chpldoc,
+  // or if we stop before codegen with --stop-after-pass or --no-codegen
```

Comment:
Sweet!  I'd been curious whether moving this check up considered this case.  Great to see that it did, thanks!